### ISSUE TITLE
Add width of columns option

### ISSUE THREAD

**Issue by JulianaGalvisN** [datasketch/app-viz-tables#10]
When I have a lot of text in a row, the column is very short and I can't change the width to visualize the table.

**Comment by brazadas** [comment]
ardilla, there is an option for defining the width of the whole table, another one for deciding if overflowed text should be wrapped (many lines of text) an another one for manually resizing columns; what do you think about this? is it enough for what you look?
![Screenshot from 2020-07-31 14-38-56](https://user-images.githubusercontent.com/19413995/89071175-ca5f5d00-d33b-11ea-8cd3-91b5b644c75a.png)
@JulianaGalvisN

**Comment by JulianaGalvisN** [comment]
No, digo que podría haber una opción para colocar el ancho de las columnas como uno quiera, además de permitir cambiar el ancho manualmente.
Por ejemplo esta tabla tiene columnas de un ancho muy pequeño, pero como son tantas columnas es mucho más difícil hacerlo manualmente 
![Captura de pantalla 2020-08-10 a la(s) 4 49 18 p  m](https://user-images.githubusercontent.com/43793348/89835009-7f0e3100-db29-11ea-9209-2da140f8e9a1.png)
